portx/modules/ROOT/pages/_partials/documents-view.adoc

```diff
@@ -22,28 +22,28 @@ Each row has 6 columns:
 === Document Detail Pane (Right)
 This pane is hidden when no document is selected. It contains:
 
-* Document ID icon:plus-square-o[role="blue"]
-* Document Definiton ID icon:plus-square-o[role="blue"]
-* Standard
-* Version
-* Message Type
-* From
-* To
-* Date
-* Duplicate Quiry
-* Message Payload icon:share[role="blue"]
-* Partnership ID icon:plus-square-o[role="blue"]
-
-==== Action Icons
-
-* The blue copy icon icon:plus-square-o[role="blue"] allows you to copy information to your clipboard
-* The blue share icon icon:share[role="blue"] opens a detail pane that shows the complete message payload.
+* *Document ID* icon:plus-square-o[role="blue"]: Unique copyable document identification.
+* *Document Definiton ID*: Unique copyable document definition identification. icon:plus-square-o[role="blue"]
+* *Standard*: The document standard used.
+* *Version*: The version of the document.
+* *Message Type*: The type/reason for the message.
+* *From*: Who the document is from.
+* *To*: Who the document is to.
+* *Date*: Date the document was sent.
+* *Duplicate Quiry*: Indicating if a document is a duplicate.
+* *Message Payload*: icon:share[role="blue"]Link to payload url and payload content
```

Comment:
I probably would not have ended these definitions with a period, reasoning that they are sentence fragments; but, thinking about how definitions appear in a dictionary, I think they always end in periods, regardless. So, keep these periods, but add one here to be consistent.